apiserver/facades/client/subnets/subnets_test.go

```diff
@@ -321,320 +321,6 @@ func (s *SubnetsSuite) TestAllSpacesFailure(c *gc.C) {
 	)
 }
 
-func (s *SubnetsSuite) TestAddSubnetsParamsCombinations(c *gc.C) {
```

Comment:
This is a duplicate of networkingcommon_test.TestAddSubnetsParamsCombinations().  https://github.com/juju/juju/blob/develop/apiserver/common/networkingcommon/subnets_test.go#L261